README.md

```diff
@@ -2,40 +2,63 @@
 
 Backend of the ScholarX project
 
-### Setting up the project for development
+## Setting up the project for development
 
-**Prerequisites**
+###Prerequisites
 * Java 
 * Maven
 * MySQL
 * Linkedin Social Login App
 * Gmail Account with an App Password 
 
-**Setup Linkedin Social Login App**
+###Setup Linkedin Social Login App
```

Comment:
`###Setup Linkedin Social Login App` I think there needs to be space-like `### Setup Linkedin Social Login App`, can you check again?

Reply:
Sure. My bad